docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -0,0 +1,49 @@
+.hb-table-wrap {
+  margin: auto;
+  overflow-x: auto;
+}
+
+table {
+  $border-color: lighten($su-color-driftwood, 15%);
+  // DO NOT set display: block; on a table
+  // Tables need display: table; in order to semantically be a table
+  // display: block; will negatively affect screen reader a11y
+  width: 100%;
+  border: 1px solid $border-color;
+  border-collapse: collapse;
+
+  font-size: 0.8rem;
+  text-align: left;
+
+  @include grid-media('sm') {
+    font-size: 1rem;
+  }
+
+  th, td {
+    border: 1px solid $border-color;
```

Comment:
we're trying not to use pixels, so if you dont mind converting to rem that'd be awesome!

Reply:
In the case of border thickness, I think we should stick to pixels. REMs allow sizes to change with scaling, but I don't see us finding any benefits in scaling border widths when a user zooms into a page.